apps/frontend/src/network/profile/__tests__/Routes.test.tsx

```diff
@@ -50,135 +41,143 @@ const renderProfile = async (profileId = '42', waitForLoad = true) => {
   );
   return result;
 };
-describe('without team', () => {
-  // fetch user by code request
-  beforeEach(() => {
-    nock.cleanAll();
-    nock(API_BASE_URL, {
-      reqheaders: { authorization: 'Bearer token' },
-    })
-      .get('/users/42')
-      .reply(200, user);
-  });
 
-  it('renders a loading indicator', async () => {
-    const { getByText } = await renderProfile('42', false);
+it('initially renders a loading indicator', async () => {
+  const { getByText } = await renderProfile();
 
-    const loadingIndicator = getByText(/loading/i);
-    expect(loadingIndicator).toBeVisible();
+  const loadingIndicator = getByText(/loading/i);
+  expect(loadingIndicator).toBeVisible();
 
-    await waitForElementToBeRemoved(loadingIndicator);
-  });
+  await waitForElementToBeRemoved(loadingIndicator);
+});
 
-  it('renders a member information', async () => {
-    const { findByText } = await renderProfile();
-    expect((await findByText(user.displayName)).tagName).toBe('H1');
+it('renders the personal info', async () => {
+  const { findByText } = await renderProfile({
+    ...createUserResponse(),
+    displayName: 'Someone',
   });
+  expect((await findByText('Someone')).tagName).toBe('H1');
+});
 
-  it('renders the research member content', async () => {
-    const { findByText } = await renderProfile();
-    expect(await findByText('Open Questions')).toBeVisible();
+it('by default renders the research tab', async () => {
+  const { findByText } = await renderProfile({
+    ...createUserResponse(),
+    questions: ['What?'],
   });
+  expect(await findByText('What?')).toBeVisible();
+});
+
+it('renders the 404 page for a missing user', async () => {
+  const { findByText } = await renderProfile(
+    {
+      ...createUserResponse(),
+      id: '42',
+    },
+    { routeProfileId: '1337' },
+  );
+  expect(await findByText(/sorry.+page/i)).toBeVisible();
 });
 
+// TODO improve test quality
```

Comment:
Would have loved to give this component more proper tests, but yeah this is already huge...